### PR TITLE
add options to control to otel resources

### PR DESCRIPTION
**Pull request by hobord**

### CODE REVIEW

File: cluster/variables.tf [hunk 1]
```diff
@@ -35,3 +35,32 @@ variable "permissions_boundary" {
...
+    check_interval = string
+    limit_percentage: number
+    spike_limit_percentage: number
+
+  })
+  default          = {
+    check_interval: "1s"
+    limit_percentage: 70
+    spike_limit_percentage: 30
+  }
```
Comment (eyalzek) [inline cluster/variables.tf]: I'm confused about the mixing of colon and equal signs

**Comment by eyalzek** [comment]
Did someone run into issues, or why is this change suggested?
Reading through [1] it sounds like this should be done on top of configuring `GOMEMLIMIT` and probably memory requests/limits since this is running on k8s.
But in this case the resource requirements are optional and the memory limiter has defaults set, so I'm slightly confused.

[1] https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md#best-practices

**Comment by hobord** [comment]
I added the option to set the GOMEMLIMIT env variable too...

**Review by eyalzek — CHANGES_REQUESTED** [review]
Update readme [1] to reflect new variables. Might also be useful to add a `description` rather than inline comments

[1] https://github.com/doitintl/terraform-eks-lens/blob/main/cluster/README.md

**Review by eyalzek — APPROVED** [review]

**Review by lauraseidler — APPROVED** [review]

**Pull request merged**